priv/repo/migrations/20221219043118_create_keywords.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Crawler.Repo.Migrations.CreateKeywords do
+  use Ecto.Migration
+
+  def change do
+    create table(:keywords) do
+      add :name, :string, null: false
+      add :status, :string, null: false
+      add :user_id, references(:users, on_delete: :delete_all), null: false
+
+      timestamps()
+    end
+
+    create index(:keywords, [:user_id])
+    create unique_index(:keywords, [:name, :user_id])
```

Comment:
I don't think we need this @vnntsu as the user could able to re-upload the same keyword :D like today I search for iPhone 14, tomorrow I'm still able to search for iPhone 14 😄

Reply:
@andyduong1920  Yeah, I also did the same idea but in a different way. I will not allow inserting the same record in the DB; however, I modify the status to new, so the user can keep searching for the keyword again. What do you think?

Reply:
yeah, just realize your idea when reviewing the other files, I added my opinion on https://github.com/vnntsu/elixir-ic/pull/36#discussion_r1056030542